lib/presentation/util/router/app_navigation.dart

```diff
@@ -42,6 +42,10 @@ class AppNavigation {
         .pushNamed(routeName, arguments: RouteSettings(arguments: arguments));
   }
 
+  Future pushRouteAndRemoveAll(Route route) {
+    return navigatorKey.currentState.pushAndRemoveUntil(route, (Route<dynamic> route) => false);
```

Comment:
Can you explain to me about that?

Reply:
This is not needed. I forgot to delete it

Reply:
sr, why don't u use existed method `pushAndRemoveAll(String routeName)` ?

Reply:
Tôi đã xử lý việc tạo ra một PageRoute trong SharedSpaceDocumentNavigatorWidgetState nên tôi đã dùng pushAndRemoveUntil để xử lý luôn cái route đó. Nên tôi nghĩ không cần phải sử dụng pushNamedAndRemoveUntil vì nó không được gọi từ một nơi khác.